spec/models/carted_product_spec.rb

```diff
@@ -22,5 +22,13 @@
         expect(customer.carted_products.count).to eq(1)
       end
     end
+    context "logged in" do
```

Comment:
@cdanmar I am not sure if we need this context? Cool for now, maybe correct in a future request.